doc/oper.xml

```diff
@@ -1740,3 +1740,42 @@ gap> DigraphDijkstra(D, 1, 2);
 </Description>
 </ManSection>
 <#/GAPDoc>
+
+
+<#GAPDoc Label="ModularProduct">
```

Comment:
Please remove the stuff about `ModularProduct` since it was already merged in #354.